### ISSUE TITLE
Creating a new windows server

### ISSUE THREAD

**Issue by JasperLD** [EQEmu/Server#2794]
Hi,

When I create a new server I have a problem with Pearl.exe
I think it has to do with the ruleset values. 

I just want to say that the installer for a windows server doesn't work atm unless I do something wrong.

![image](https://user-images.githubusercontent.com/58681025/214662099-4bbe431f-8c46-4022-8bb6-0462db81e0d4.png)

**Comment by fryguy503** [comment]
Set your database version back about 10 versions and run world so it will re-run the schema updates.

**Comment by jchasta1n** [comment]
I had this same problem and fixed it same way as fryguy.  Use Heidi and there is a table in the database called db_version.  I was way conservative and set both all the way back to 10 and then let it rerun all of the updates.  This is a recent problem, so going so far back isn't necessary but I'm confident I got everything.  I used eqemu_server.pl to perform the updates after resetting the versions just so I could watch everything run.  The DB updates are obviously found in the database sub-menu.
